gctf/__init__.py

```diff
@@ -43,8 +43,8 @@ class Plugin(pyworkflow.em.Plugin):
 
     @classmethod
     def _defineVariables(cls):
-        cls._defineEmVar(GCTF_HOME, 'gctf-1.06')
-        cls._defineVar(GCTF, 'Gctf-v1.06_sm_20_cu8.0_x86_64')
+        cls._defineEmVar(GCTF_HOME, 'gctf-1.18')
```

Comment:
@azazellochg I still don't understand why we are switching to 1.18 here, if this version is "special" for VPP. As far as know, VPP is not the most used case for CTF estimation.

Reply:
I asked Kai and in principle, it is exactly the same as 1.06 for non-VPP data. Except missing refinement and movie/validations options. Don't know... maybe we could leave it as is